### PR TITLE
Contents

### PR DESCRIPTION
**Pull request by jordan2lee**
Two analysis sections. Placed limits to query for reasonable app load time. 

TODO: cache data and remove limits

### CODE REVIEW

**Comment by kellrott** [comment]
Because `render_callback` invokes the `get_df` and `get_umap` it always redoes the calculation, and the points can move. If someone changes the label, those should stay the same, and the colors of the dots should change. 
You may want to look at Dash State data ( https://dash.plotly.com/sharing-data-between-callbacks ) to store the umap locations that are calculate when someone changes cancer type, but then only have the dot labeling change when a different feature is selected

File: README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# BMEG Web App
+
+Run app `python -m bmeg_app`
```
Comment (bwalsh) [inline README.md:3]:
```
$ python -m bmeg_app
Traceback (most recent call last):
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/runpy.py", line 174, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/Users/walsbr/bmeg-dash/bmeg_app/__main__.py", line 2, in <module>
    from bmeg_app.app import app
  File "bmeg_app/app.py", line 2, in <module>
    import dash
ImportError: No module named dash
```

Suggest including  `venv setup` instructions.

**Review by bwalsh — COMMENTED** [review]
There is a large amount of duplicated  code.  Need to extract common code into helper functions or super class.

```
$ comm app_about.py app_drugresp.py | wc -l
     332
$ comm app_drugresp.py app_tcga.py | wc -l
     231
```

**Comment by jordan2lee** [comment]
these commits will be incorporated into PR from branch `report`

**Pull request closed without merging**